recipes/weeder/post-link.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Downloading and installing Weeder frequency files..."
+
+# Directory to store FreqFiles
+TGT="$PREFIX/share/weeder2"
+
+# Cargo port url
+CARGO_URL=http://depot.galaxyproject.org/software/
+
+# Download and unpack
+TARBALL=${PKG_NAME}_${PKG_VERSION}_src_all.tar.gz
+curl -L ${CARGO_URL}/${PKG_NAME}/$TARBALL -o $TGT/$TARBALL
```

Comment:
@simonvh have you tested this locally? Is `PKG_NAME` available at install time? This would be great, was not aware of this.